com/ConvertTrimble.py

```diff
@@ -15,23 +15,30 @@
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Script to convert T0x files to RINEX')
+    parser = argparse.ArgumentParser(
+        description='Script to convert T0x files to RINEX')
 
     parser.add_argument('path', type=str, nargs=1, metavar='[path to dir]',
                         help="Path to directory with T0x files")
 
     parser.add_argument('path_out', type=str, nargs=1, metavar='[path to dir]',
-                        help="Path to directory with resulting RINEX (a folder with station name will be created)")
+                        help='''Path to directory with resulting RINEX
+                             (a folder with station name will be created)''')
 
     parser.add_argument('-stnm', '--station_name', type=str, default='dftl',
                         help="Name of the station to form that RINEX files")
 
     parser.add_argument('-ant', '--antenna_name', type=str, nargs='+',
-                        action=required_length(2, 3), metavar='{atx_file} {antenna_name} [SN]', default=None,
-                        help="Replace the antenna name/type in the raw file with name provided in {antenna_name}. "
-                             "Antenna has to exist in ATX file provided in {atx_file}. Radome will be set to NONE by "
-                             "default. Antenna name can have wildcards but a single match is expected (otherwise an "
-                             "exception will be raised). Optionally, provide the antenna serial number.")
+                        action=required_length(2, 3),
+                        metavar='{atx_file} {antenna_name} [SN]', default=None,
```

Comment:
approving this, although it probably makes most sense in this context for `default=None` to be on it's own line here given that the `action`, `metavar`, and `help` parameters are all starting on separate lines...